workers/generic-worker/multiuser_posix.go

```diff
@@ -36,12 +36,12 @@ func platformFeatures() []Feature {
 
 func deleteDir(path string) error {
 	log.Print("Removing directory '" + path + "'...")
-	err := host.Run("/usr/bin/sudo", "/bin/chmod", "-R", "u+w", path)
+	err := host.Run("/bin/chmod", "-R", "u+w", path)
```

Comment:
will it cause any issues for the non docker-compose workers? Or will this always be run with the correct permissions?